modules/opta-k8s-service-helm/templates/crons.yaml

```diff
@@ -0,0 +1,62 @@
+{{- range $index, $cron := .Values.crons }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "k8s-service.fullname" $ }}-{{ $index }}
+  labels:
+    tags.datadoghq.com/service: {{ include "k8s-service.serviceName" $ }}-{{ include "k8s-service.namespaceName" $ }}
+    tags.datadoghq.com/version: {{ $.Values.version | quote }}
+  {{- include "k8s-service.labels" $ | nindent 4 }}
+  namespace: {{ include "k8s-service.namespaceName" $ }}
+spec:
+  schedule: {{ $cron.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: "Never"
+          {{- if ne ( len $.Values.tolerations ) 0 }}
+          tolerations:
+          {{ range $toleration :=  $.Values.tolerations }}
+          - key: {{ $toleration.key | quote }}
+            value: {{ default "opta" $toleration.value | quote }}
+            effect: {{ default "NoSchedule" $toleration.effect | quote }}
+          {{ end }}
+          {{- end }}
+          containers:
+            - name: cron
+              image: "{{ $.Values.image }}"
+              imagePullPolicy: Always
+              command:
+              {{ range $command := $cron.commands }}
+              - {{ $command | quote }}
+              {{ end }}
+              env:
+              - name: RDS_CA_PATH # This is the path to the public key for the docdb tls
```

Comment:
Why is this specific RDS_CA_PATH a special case to be defined inside the template; separate from the other envVars?

Reply:
needed for documentdb from way back in the day. Super annoying and we might want to consider removing it lol